microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -77,6 +77,26 @@ When you visit a webpage and submit a form, Microsoft Edge sends information abo
 
 In **Guest** mode, autofill is not available and new autofill entries aren't added.  For **InPrivate** mode, Microsoft Edge offers autofill entries, but new autofill entries aren't added.
 
+<!-- ====================================================================== -->
+## Bing Chat
+
+Bing chat is an AI-enhanced web search experience. It supports users by summarizing web search results and providing a chat experience. Users can also generate creative content, such as poems, jokes, and letters. The new AI-enhanced Bing runs on a variety of advanced technologies from Microsoft and OpenAI, including GPT-4, a cutting-edge large language model (LLM) from OpenAI. 
+
+When a user enters a prompt in the new Bing, the prompt, recent conversation history, the meta prompt, page context, and top search results are sent as inputs to the LLM. The model generates a response using the page context, user’s prompt, and recent conversation history to contextualize the request, the meta prompt to align responses with Microsoft AI Principles and user expectations, and the search results to ground responses in existing, high-ranking content from the web.  
```

Comment:
```suggestion
When a user enters text in a prompt in the new Bing, the prompt, recent conversation history, the meta prompt, page context, and top search results are sent as inputs to the LLM. The model generates a response using the page context, user’s prompt, and recent conversation history to contextualize the request, the meta prompt to align responses with Microsoft AI Principles and user expectations, and the search results to ground responses in existing, high-ranking content from the web.  
```
* "text in" - not sure if this is always text, or a variety of UI controls settings/values.